### ISSUE TITLE
Is there an inline mode

### ISSUE THREAD

**Issue by jeandat** [dhutaryan/ngx-mat-timepicker#62]
Didn't tried it but I tested the demo and I find it very clean. I'm currently looking at what exists on github for this type of components. Is it possible to use it directly without any trigger or input?

**Comment by dhutaryan** [comment]
Hey, thank you for your question. It's possible to use directly but one mode at the same time only. You can look at it on the demo already.
```
<mat-clock-dials [selected]="date"></mat-clock-dials>
```
```
<mat-time-inputs [(selected)]="date"></mat-time-inputs>
```
I've fixed some issues for that as well. Fixes will be available in `12.1.0`

**Comment by dhutaryan** [comment]
Think I can close this issue. If you have any other questions or find bugs feel free to create a new issue.
